packages/vsf-storyblok-module/urlExtend.ts

```diff
@@ -40,16 +41,20 @@ const forCategory = async ({ dispatch }, { url }) => {
   }
 }
 
-const forStory = async ({ dispatch }, { url: fullSlug }) => {
+const forStory = async ({ dispatch }, { url: fullSlug, params }) => {
+  const result = {
+    name: 'storyblok',
+    params: {
+      // TODO: Why does this need to be here?
+      slug: 'storyblok'
+    }
+  }
+  if (params._storyblok) {
+    return result
+  }
   const story = await dispatch(`${KEY}/loadStory`, { fullSlug }, { root: true })
```

Comment:
Shouldn't we dispatch a `loadDraftStory` in the case we're in editor mode?

And have the logic be as it is, i.e. redirect if it can't find a story or a draft story in the case of in editor mode?

Reply:
@zimme unless I'm mistaken `loadDraftStory` would be run on mount anyways. I just did this so you wouldn't get "Page not found" when creating a new story.

But maybe we should put the init logic here instead?

Reply:
If we're fetching stories during routing now to be able to redirect when there's no category, product or story based on the url, maybe we should just add the story fetched here into the store and don't fetch it again during mount on components extending storyblok/page?